tgui/packages/tgui/styles/interfaces/PanDEMIC.scss

```
@@ -0,0 +1,31 @@
+.symptoms-table {
+  border-collapse: separate;
+  border-spacing: 0 0.5ex;
```

```suggestion
  border-spacing: 0 0.5ex;
```
There's also ex, which is not used anywhere in the TGUI

---

I personally still don't like the use of ex instead of em